tools/devops/automation/templates/packages/build.yml

```diff
@@ -323,6 +323,20 @@ steps:
     artifactName: package-internal
   continueOnError: true
 
+# funny enough we need these profiles to build the mac tests
+- bash: |
+    set -x
+    set -e
+    rm -f ~/Library/Caches/com.xamarin.provisionator/Provisions/*p12
+    rm -f ~/Library/Caches/com.xamarin.provisionator/Provisions/*mobileprovision
```

Comment:
The `install-qa-provisioning-profiles.sh` script already does this, so these lines shouldn't be necessary.